sematic/utils/exceptions.py

```diff
@@ -208,3 +208,23 @@ class MissingPluginError(Exception):
     def __init__(self, plugin_path: str):
         message = f"Unable to find plug-in {plugin_path}. Module or class is missing."
         super().__init__(message)
+
+
+class UnsupportedError(Exception):
```

Comment:
This sounds like the error is unsupported which does make a lot of sense to the reader. Can we rename to something like `UnsupportedContextError` or something?